pkgs/development/python-modules/autobahn/default.nix

```diff
@@ -12,14 +12,14 @@ buildPythonPackage rec {
     sha256 = "2f41bfc512ec482044fa8cfa74182118dedd87e03b3494472d9ff1b5a1e27d24";
   };
 
-  # Upstream claim python2 support, but tests require pytest-asyncio which
-  # is pythn3 only. Therefore, tests are skipped for python2.
-  doCheck = isPy3k;
-  buildInputs = stdenv.lib.optionals isPy3k [ unittest2 mock pytest pytest-asyncio ];
   propagatedBuildInputs = [ six twisted zope_interface txaio ] ++
     (stdenv.lib.optional isPy33 asyncio) ++
     (stdenv.lib.optionals (!isPy3k) [ trollius futures ]);
 
+  # Upstream claim python2 support, but tests require pytest-asyncio which
+  # is python3 only. Therefore, tests are skipped for python2.
+  doCheck = isPy3k;
+  checkInputs = [ unittest2 mock pytest pytest-asyncio ];
```

Comment:
The tests do definitely support python2: https://github.com/crossbario/autobahn-python/blob/master/tox.ini.